_code-samples/nft-modular-tutorials/account-support.js

```diff
@@ -0,0 +1,207 @@
+// ******************************************************
+// ************* Get the Preferred Network **************
+// ******************************************************  
+
+function getNet() {
+  let net
+  if (document.getElementById("tn").checked) net = "wss://s.altnet.rippletest.net:51233/"
+  if (document.getElementById("dn").checked) net = "wss://s.devnet.rippletest.net:51233/"
+  return net
+} // End of getNet()
+
+// *******************************************************
+// ************* Get Account *****************************
+// *******************************************************
+
+async function getAccount() {
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  await client.connect()
+  resultField.value = `===Getting Account===\n\nConnected to ${net}.`
+  try {
+    let faucetHost = null
+    const my_wallet = (await client.fundWallet(null, { faucetHost})).wallet
+    const newAccount = [my_wallet.address, my_wallet.seed]
+    return (newAccount)
```

Comment:
There's no need for the parenthesis, as we are just returning `newAccount`. Think there's a few other places where we do this.
```suggestion
    return newAccount
```